src/pumgen.cpp

```diff
@@ -127,20 +129,24 @@ int main(int argc, char* argv[])
 			logInfo(rank) << "Using Fidap mesh";
 			meshInput = new SerialMeshFile<ParallelFidapReader>(inputFile);
 			break;
-		case 2:
+        case 2:
```

Comment:
seems like not well aligned with SR tabs :-p

Reply:
Tabs = evil :)

Reply:
Maybe we can use pumgen as a "playground" to test out an enforced coding style.